base/exports.jl

```diff
@@ -1111,6 +1111,7 @@ export
     atexit,
     clipboard,
     exit,
+    isexiting,
```

Comment:
This strikes me as something we shouldn't be exporting. I guess it's an advanced feature, but it feels odd that the user might need to write code that has to know whether it's exiting or not.